### PR TITLE
Material repair config attempt.

### PR DESCRIPTION
**Pull request by SpGilbert**
I am running into problems of it breaking crafttweaker even if its just atop.zs. While aether.zs also breaks it. I added all diamond, ruby, alxeandrite, saphhire, zanite and gravitite gear to it.
[debug-2.log.gz](https://github.com/Rebirth-of-the-Night/Rebirth-Of-The-Night/files/9095957/debug-2.log.gz)
[2022-07-09-5.log.gz](https://github.com/Rebirth-of-the-Night/Rebirth-Of-The-Night/files/9095962/2022-07-09-5.log.gz)
[crash-2022-07-09_21.15.18-client.txt](https://github.com/Rebirth-of-the-Night/Rebirth-Of-The-Night/files/9095967/crash-2022-07-09_21.15.18-client.txt)
[crash-2022-07-09_21.27.30-client.txt](https://github.com/Rebirth-of-the-Night/Rebirth-Of-The-Night/files/9095971/crash-2022-07-09_21.27.30-client.txt)

### CODE REVIEW

**Comment by SpGilbert** [comment]
kind of a mess due to originaly putting it in the vanillanvil repair config. But I was told to split it to aether.zs and atop.zs later.

File: config/vanillaanvilrepair.cfg [hunk 1]
```diff
@@ -5,7 +5,7 @@ general {
     # repaireditem1,repaireditem2[...];material1,material2[...]
     # (you can use ore:item for OreDictionary entry (only for materials))
     S:repairList <
-     >
+>
```
Comment (democat3457) [inline config/vanillaanvilrepair.cfg:8]: revert this change

File: scripts/aether.zs [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+addRepairEntry(zaniteItems, <contenttweaker:zanite_ingot>);
+
+var gravititeItems = <aether_legacy:gravitite_helmet> | <aether_legacy:gravitite_chestplate> | <aether_legacy:gravitite_leggings> | <aether_legacy:gravitite_boots. | <aether_legacy:gravitite_sword> | <atop:amber_pickaxe> |  <atop:amber_shovel> | <atop:amber_axe> | <spartancompat:battleaxe_gravitite> | <spartancompat:boomerang_gravitite> | <spartancompat:dagger_gravitite> | <spartancompat:glaive_gravitite> | <spartancompat:greatsword_gravitite> | <spartancompat:halberd_gravitite> | <spartancompat:hammer_gravitite> | <spartancompat:javelin_gravitite> | <spartancompat:katana_gravitite> | <spartancompat:lance_gravitite> | <spartancompat:longbow_gravitite> | <spartancompat:longsword_gravitite> | <spartancompat:mace_gravitite> | <spartancompat:pike_gravitite> | <spartancompat:rapier_gravitite> | <spartancompat:saber_gravitite> | <spartancompat:spear_gravitite> | <spartancompat:staff_gravitite> | <spartancompat:throwing_axe_gravitite> | <spartancompat:throwing_knife_gravitite> | <spartancompat:warhammer_gravitite>;
```
Comment (democat3457) [inline scripts/aether.zs:17]:
```suggestion
var gravititeItems = <aether_legacy:gravitite_helmet> | <aether_legacy:gravitite_chestplate> | <aether_legacy:gravitite_leggings> | <aether_legacy:gravitite_boots> | <aether_legacy:gravitite_sword> | <atop:amber_pickaxe> |  <atop:amber_shovel> | <atop:amber_axe> | <spartancompat:battleaxe_gravitite> | <spartancompat:boomerang_gravitite> | <spartancompat:dagger_gravitite> | <spartancompat:glaive_gravitite> | <spartancompat:greatsword_gravitite> | <spartancompat:halberd_gravitite> | <spartancompat:hammer_gravitite> | <spartancompat:javelin_gravitite> | <spartancompat:katana_gravitite> | <spartancompat:lance_gravitite> | <spartancompat:longbow_gravitite> | <spartancompat:longsword_gravitite> | <spartancompat:mace_gravitite> | <spartancompat:pike_gravitite> | <spartancompat:rapier_gravitite> | <spartancompat:saber_gravitite> | <spartancompat:spear_gravitite> | <spartancompat:staff_gravitite> | <spartancompat:throwing_axe_gravitite> | <spartancompat:throwing_knife_gravitite> | <spartancompat:warhammer_gravitite>;
```

**Review by democat3457 — CHANGES_REQUESTED** [review]

**Comment by SpGilbert** [comment]
thanks for the fixes.

**Comment by democat3457** [comment]
> thanks for the fixes.

Can you actually implement those fixes? Those are just comments...

**Comment by democat3457** [comment]
Also, those crashes in the description are unrelated

**Comment by SpGilbert** [comment]
implemented both.

**Comment by democat3457** [comment]
syntax-wise, looks good to me

@TheNelston can you read it over to make sure all the items are mapped to the right repair item

**Pull request merged**